docs/types_distributedExecutionHttp.js.html

```
@@ -0,0 +1,105 @@
+/**
+ * HTTP surface for distributed execution: manifest and epoch status for operators.
+ * Binds routes on a {@link FabricHTTPServer} via `_addRoute` (same pattern as Hub services).
+ */
+const merge = require('lodash.merge');
+const Service = require('@fabric/core/types/service');
+
+class FabricDistributedExecutionHTTP extends Service {
+  /**
+   * @param {Object} [settings]
+   * @param {string} [settings.basePath='/services/distributed']
+   * @param {() => object|Promise&lt;object>} [settings.getManifest] — JSON manifest (see `DistributedExecution.parseDistributedManifestV1`)
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)